GoogleDataTransport/GDTCORLibrary/Internal/GDTCOREventDropReason.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+// TODO(nickcooke): Document.
+typedef NS_ENUM(NSInteger, GDTCOREventDropReason) {
```

Comment:
is this part of the proto defined in the design doc? In that case, are we supposed to use nanopb to present them?

Reply:
Yes, we will use nanopb to represent them but I defined this enum so I have a native objc type for passing to API's like: 

https://github.com/google/GoogleDataTransport/blob/8e1828d0b9d6d85b1be4bd6edf060ecb14c546fa/GoogleDataTransport/GDTCORLibrary/GDTCORMetricsController.m#L39-L41

I will add the proto in future PR.